README.md

```diff
@@ -1 +1,23 @@
-# opentelemetry-collector-operator
+# OpenTelemetry Collector Operator for Machines
+
+[![CharmHub Badge](https://charmhub.io/opentelemetry-collector/badge.svg)](https://charmhub.io/opentelemetry-collector)
+[![Release](https://github.com/canonical/opentelemetry-collector-operator/actions/workflows/release.yaml/badge.svg)](https://github.com/canonical/opentelemetry-collector-k8s-operator/actions/workflows/release.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
+
+This repository contains the source code for a Charmed Operator that drives [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector), a vendor-agnostic way to receive, process and export telemetry data, on machines (LXD, MAAS, etc).
+
+## Usage
+
+Assuming you have access to a bootstrapped Juju controller, you can:
+
+```bash
+$ juju deploy opentelemetry-collector
+```
+
+## Snap
+
+This charm, by default, uses the [Opentelemetry Collector snap](https://github.com/canonical/opentelemetry-collector-snap/).
```

Comment:
```suggestion
This charm uses the [Opentelemetry Collector snap](https://github.com/canonical/opentelemetry-collector-snap/).
```

There is now way to change that, mentioning it's a "default" subtly implies it is.